### PR TITLE
Added Integration Foregoing 1.12 CraftTweaker documentation

### PR DESCRIPTION
**Pull request by JackyyTV**
- Added documentation for Washing Factory, Fermentation Station and Fluid Sieving Machine.

### CODE REVIEW

File: docs/Mods/Integration_Foregoing/FermentationStation.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+```
+
+### Adding
+
+The first and second parameters are the input and output fluids respectively.
+
+The example shown below turns 1mB of Lava into 1mB of Blazing Pyrotheum:
+
+```zenscript
+FermentationStation.add(ILiquidStack fluidInput, ILiquidStack fluidOutput);
```
Comment (jaredlll08) [inline docs/Mods/Integration_Foregoing/FermentationStation.md:16]: Put this in a comment please, just so people know that it isn't a line that is actually meant to work

File: docs/Mods/Integration_Foregoing/FermentationStation.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+FermentationStation.add(<liquid:lava> * 1, <liquid:pyrotheum> * 1);
+```
+
+### Removing
+
+Put the input fluid in the parameter to remove the recipe for that specific fluid, has to match the fluid amount in the recipe:
+
+```zenscript
+FermentationStation.remove(ILiquidStack fluidInput);
```
Comment (jaredlll08) [inline docs/Mods/Integration_Foregoing/FermentationStation.md:26]: Put this in a comment please, just so people know that it isn't a line that is actually meant to work

File: docs/Mods/Integration_Foregoing/FluidSievingMachine.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+```
+
+### Adding
+
+The first parameter is the input fluid. The second parameter is the output item. The third parameter is the input item used with the liquid.
+
+The example shown below turns a piece of Cobblestone and 500mB of Blazing Pyrotheum into a piece of Obsidian:
+
+```zenscript
+FluidSievingMachine.add(ILiquidStack fluidInput, IItemStack itemOutput, IItemStack sieveItemInput);
```
Comment (jaredlll08) [inline docs/Mods/Integration_Foregoing/FluidSievingMachine.md:16]: Put this in a comment please, just so people know that it isn't a line that is actually meant to work

File: docs/Mods/Integration_Foregoing/FluidSievingMachine.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+FluidSievingMachine.add(<liquid:pyrotheum> * 500, <minecraft:obsidian>, <minecraft:cobblestone>);
+```
+
+### Removing
+
+Put the output item in the parameter to remove the recipe for that specific item:
+
+```zenscript
+FluidSievingMachine.remove(IItemStack itemOutput);
```
Comment (jaredlll08) [inline docs/Mods/Integration_Foregoing/FluidSievingMachine.md:26]: Put this in a comment please, just so people know that it isn't a line that is actually meant to work

File: docs/Mods/Integration_Foregoing/WashingFactory.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+```
+
+### Adding
+
+The first parameter is the input item, and it has to be an OreDict string. The second and third parameters are the input and output fluids respectively.
+
+The example shown below turns a piece of Gold Ingot and 500mB of Water into 500mB of Lava:
+
+```zenscript
+WashingFactory.add(String oreDictInput, ILiquidStack fluidInput, ILiquidStack fluidOutput);
```
Comment (jaredlll08) [inline docs/Mods/Integration_Foregoing/WashingFactory.md:16]: Put this in a comment please, just so people know that it isn't a line that is actually meant to work

File: docs/Mods/Integration_Foregoing/WashingFactory.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+WashingFactory.add("ingotGold", <liquid:water> * 500, <liquid:lava> * 500);
+```
+
+### Removing
+
+Put an OreDict string in the parameter to remove the recipe for that specific OreDict:
+
+```zenscript
+WashingFactory.remove(String oreDictInput);
```
Comment (jaredlll08) [inline docs/Mods/Integration_Foregoing/WashingFactory.md:26]: Put this in a comment please, just so people know that it isn't a line that is actually meant to work

**Review by jaredlll08 — CHANGES_REQUESTED** [review]

**Comment by jaredlll08** [comment]
You commit didn't fix the issue, right now a user will just copy that code block and paste it into the script, and it isn't going to work.

Ideally it should look like:
```zenscript
// FermentationStation.add(ILiquidStack fluidInput, ILiquidStack fluidOutput);

FermentationStation.add(<liquid:lava> * 1, <liquid:pyrotheum> * 1);
```

So users can copy it, and know what param is what.

**Comment by JackyyTV** [comment]
Sorry about that, should be fixed now.

**Comment by jaredlll08** [comment]
Thanks!

**Pull request merged**